src/Pact/Types/Typecheck.hs

```diff
@@ -250,7 +250,6 @@ instance Pretty t => Pretty (Fun t) where
     , ""
     ]
 
-
```

Comment:
This file is whitespace-only change, please revert

Reply:
:+1: